tests/framework/extensions/clusters/aks/resources/create_nodepool.go

```diff
@@ -0,0 +1,34 @@
+package resources
+
+import (
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+)
+
+// CreateNodePool is a helper function that creates a node pool.
+func CreateNodePool() (*management.AKSNodePool, error) {
+	autoScaling := true
+	diskSize := int64(120)
```

Comment:
An alternative to hardcoding a default value here, would be to pull these values from the AKSNodePool struct in aks_cluster_config.go.  diskSize, maxCount, maxPods, minCount, nodeCount are already defined there as int64, so it would also avoid this initial type conversion when declaring these.

Reply:
Similar to my comment [below](https://github.com/rancher/rancher/pull/39793#discussion_r1042421079).

Reply:
I agree with Josh, let's not hard code values

Reply:
Thank you both, I'll rework this so that we simply use the values defined in the config file as opposed to hardcoding here.